lib/mpl_toolkits/mplot3d/axes3d.py

```diff
@@ -1593,7 +1593,11 @@ def plot_surface(self, X, Y, Z, *args, norm=None, vmin=None,
             Bounds for the normalization.
 
         shade : bool
-            Whether to shade the face colors.
+            Whether to shade the facecolors. Defaults to True. Shading is
+            always disabled when cmap is specified
+
+        lightsource : LightSource
+            The lightsource to use when `shade` is True
```

Comment:
would be nice if this referenced the LightSource api docs since this isn't a common thing.